### PR TITLE
Functionality for counting log file msgs

### PR DESCRIPTION
**Pull request by Georacer**
Using pure Matlab ways

### CODE REVIEW

**Comment by Georacer** [comment]
This should deal with #6.

File: Ardupilog.m [hunk 1]
```diff
@@ -13,6 +13,7 @@
         % platform
         % version
         % bootTime
+        numMsgs
```
Comment (Georacer) [inline Ardupilog.m:16]: Added a new public property, I think it's useful

File: Ardupilog.m [hunk 2]
```diff
@@ -42,13 +43,42 @@
             % THE MAIN CALL: Begin reading specified log file
             readLog(obj);
         end
+        
+        function [] = countMsgs(obj)
+            if obj.fileID==-1
+                error('Log file should be open upon call');
+            end
+            
+            readsize = 1024*1024;
```
Comment (Georacer) [inline Ardupilog.m:52]: This block read size (1M) parses the log file almost instantly

File: Ardupilog.m [hunk 3]
```diff
@@ -42,13 +43,42 @@
...
+            indices = [];
+            carry = [];
+                        
+            while feof(obj.fileID)==0
+                batch = [carry fread(obj.fileID,readsize)'];
+                if isempty(batch)
+                    break
+                end
+                if batch(end)==163
+                    carry = 163;
```
Comment (Georacer) [inline Ardupilog.m:63]: If the 1nd byte of the start code lies at the end of the block, append it in front of the next block

**Comment by Georacer** [comment]
I should add that I haven't tested this code for correctness. Just observed reasonable results.

**Comment by hunt0r** [comment]
I think this looks great! I'm merging.

**Pull request merged**